src/traffic_cop/traffic_cop_util.cpp

```diff
@@ -25,7 +26,13 @@ std::unique_ptr<planner::AbstractPlanNode> TrafficCopUtil::Optimize(
     std::unique_ptr<optimizer::AbstractCostModel> cost_model, const uint64_t optimizer_timeout) {
   // Optimizer transforms annotated ParseResult to logical expressions (ephemeral Optimizer structure)
   optimizer::QueryToOperatorTransformer transformer(accessor, db_oid);
-  auto logical_exprs = transformer.ConvertToOpExpression(query->GetStatement(0), query);
+  common::ManagedPointer<parser::SQLStatement> query_statement = query->GetStatement(0);
+
+  if (query_statement->GetType() == parser::StatementType::EXPLAIN) {
```

Comment:
Can you add a comment what's happening here? Presumably you're extracting the underlying statement for optimization?